src/otx/core/data/entity/base.py

```diff
@@ -157,10 +157,6 @@ def collate_fn(cls, entities: list[T_OTXDataEntity]) -> OTXBatchDataEntity:
             msg = "collate_fn() input should include a single OTX task"
             raise RuntimeError(msg)
 
-        if not all(entity.image_type == ImageType.TV_IMAGE for entity in entities):
```

Comment:
Why do we need this change?

Reply:
Because OV IR should receive Numpy images, not torch vision tensors. This condition raised an error

Reply:
There is no `np.ndarray` type in `OTXBatchDataEntity.images`. Please make it consistent.

Reply:
I will add np.array to OTXBatchDataEntity.images, bacause it is essential to use numpy arrays for OV model

Reply:
Can we maintain the type of `OTXBatchDataEntity.images` as `tv_tensor.Image` only?
I confirmed that we don't need to pay data copy cost between `np.ndarray` and `tv_tensors.Image` conversion.
Please see the following code. It shows that they share the same pointer to the internal data buffer.

```python
import numpy as np
import torch
from torchvision.tv_tensors import Image

np_tensor = np.zeros([2, 2])
tv_image = Image(data=torch.from_numpy(np_tensor))
back_to_np = tv_image.numpy()
np_tensor += 1

print("np_tensor", np_tensor)
print("tv_image", tv_image)
print("back_to_np", back_to_np)
```

```console
np_tensor [[1. 1.]
 [1. 1.]]
tv_image Image([[[1., 1.],
        [1., 1.]]], dtype=torch.float64, )
back_to_np [[[1. 1.]
  [1. 1.]]]
```

Reply:
I used ToImage transform from torchvision. I removed Identity since it is not needed anymore. In "_customize_inputs" I added conversion from torch.tensort to np.array